core/storage/feedback/gae_models.py

```diff
@@ -299,12 +299,8 @@ def get_most_recent_message(cls, thread_id):
             GeneralFeedbackMessageModel. Last message in the thread.
         """
         thread = GeneralFeedbackThreadModel.get_by_id(thread_id)
-        if thread.message_count:
```

Comment:
If you want to get rid of this then we need a one-off job to check that historical production data always sets thread.message_count.

Reply:
Oops I deleted this when I was playing with the code and I forgot to checkout. Reverted the changes and added test for the same